components/board/client.go

```diff
@@ -185,6 +189,28 @@ func (c *client) ModelAttributes() ModelAttributes {
 	return ModelAttributes{Remote: true}
 }
 
+func (c *client) SetPowerMode(ctx context.Context, mode pb.PowerMode, duration *time.Duration) error {
+	var dur *durationpb.Duration
+	if duration != nil {
+		dur = durationpb.New(*duration)
+	}
+	_, err := c.client.SetPowerMode(ctx, &pb.SetPowerModeRequest{Name: c.info.name, PowerMode: mode, Duration: dur})
+	if err != nil {
+		// A board asked to go into an offline power mode may
+		// not be able to reliably reply to the RPC. Bury
+		// errors that look like no reply was sent.
+		if mode == pb.PowerMode_POWER_MODE_OFFLINE_DEEP {
```

Comment:
I don't think we should filter out errors at this point, it should be documented that the call may not return with succeed. Ultimately it would be preferable to return success even when going in deepsleep.

Reply:
No problem, removed.